src/lib/routes/admin-api/metrics.ts

```diff
@@ -43,10 +47,15 @@ class MetricsController extends Controller {
                 openApiService.validPath({
                     tags: ['Metrics'],
                     operationId: 'createApplication',
+                    summary:
+                        'Create an application to connect reported metrics',
+                    description:
+                        'Is used to report usage as well which sdk the application uses',
```

Comment:
What does the description describe? Is that about the created application? And what is this application? I don't know enough about it. Is it documented anywhere?

```suggestion
                    summary:
                        'Create an application to collect metrics',
                    description:
                        'Creates an application in Unleash that is used to bundle metrics. The data collected includes usage data and what SDKs are used.',
```